modules/http-load-balancer/README.md

```diff
@@ -1,78 +1,75 @@
 # HTTP(S) Load Balancer Module
```

Comment:
I think we only needed to update the README of the root repo, unless there was a service in the catalog that mapped to an individual module. With that said, this is a much nicer format overall so still good to update these as well! The only downside is the duplicated boilerplate text...

Reply:
Ah, ok. Think I misunderstood that part. Why I'm tempted to revert to the old format is that the original module documentation contains module usage instructions and already had a lot of the core concepts embedded. What do you think?

Reply:
Ah that's a good point. That also probably looks better in the terraform registry too. I am in agreement with you about reverting.